themes/10up-theme/theme.ui.json

```diff
@@ -0,0 +1,403 @@
+{
+  "version": 2,
+  "$schema": "https://schemas.wp.org/wp/5.9/theme.json",
+  "settings": {
+    "custom": {
+      "color": {
+        "base": {
+          "primary": {
+            "ultralight": "#fecaca",
+            "light": "#dc2626",
+            "medium": "#b91c1c",
+            "dark": "#991b1b"
+          },
+          "secondary": {
+            "ultralight": "#bbf7d0",
+            "light": "#16a34a",
+            "medium": "#15803d",
+            "dark": "#166534"
+          },
+          "tertiary": {
+            "ultralight": "#bfdbfe",
+            "light": "#2563eb",
+            "medium": "#1d4ed8",
+            "dark": "#1e40af"
+          },
+          "grey": {
+            "100": "#f3f4f6",
+            "300": "#d1d5db",
+            "500": "#6b7280",
+            "700": "#374151",
+            "900": "#111827"
+          },
+          "dark": "#222",
+          "light": "#FFF",
+          "alert": "#ef4444",
+          "succes": "#22c55e",
+          "disabled": "var(--wp--custom--color--base--grey--300)",
+          "outline": "rgb(0 125 250 / 60%)"
+        },
+        "text": {
+          "primary": "var(--wp--custom--color--base--dark)",
+          "on-dark": "var(--wp--custom--color--base--light)",
+          "link": "var(--wp--custom--color--base--primary--medium)",
+          "disabled": "var(--wp--custom--color--disabled)"
+        },
+        "interactive": {
+          "primary": {
+            "default": "var(--wp--custom--color--base--primary--medium)",
+            "active": "var(--wp--custom--color--base--primary--dark)",
+            "outline": "var(--wp--custom--color--base--outline)"
+          },
+          "secondary": {
+            "default": "var(--wp--custom--color--base--secondary--medium)",
+            "active": "var(--wp--custom--color--base--secondary--dark)",
+            "outline": "var(--wp--custom--color--base--outline)"
+          },
+          "tertiary": {
+            "default": "var(--wp--custom--color--base--tertiary--medium)",
+            "active": "var(--wp--custom--color--base--tertiary--dark)",
+            "outline": "var(--wp--custom--color--base--outline)"
+          }
+        }
+      },
+      "space": {
+        "relative": "1em",
+        "relative-sm": ".5em",
+        "relative-lg": "2em",
+        "base": "8px",
+        "half-x": "calc(var(--wp--custom--space--base) * .5)",
+        "default": "calc(var(--wp--custom--space--base) * 2)",
+        "3-x": "calc(var(--wp--custom--space--base) * 3)",
+        "4-x": "calc(var(--wp--custom--space--base) * 4)",
+        "5-x": "calc(var(--wp--custom--space--base) * 5)",
+        "6-x": "calc(var(--wp--custom--space--base) * 6)",
+        "7-x": "calc(var(--wp--custom--space--base) * 7)",
+        "8-x": "calc(var(--wp--custom--space--base) * 8)",
+        "9-x": "calc(var(--wp--custom--space--base) * 9)",
+        "10-x": "calc(var(--wp--custom--space--base) * 10)",
+        "inset": {
+          "squish": {
```

Comment:
This might be a language barear thing, but I'm not sure what this is suposed to mean / what it is going to be used for :)

Reply:
It is a setting to set padding on an element that is larger on the sides than on the top - like buttons, accordion headers, etc. It can get a lot more complex (see here: https://jdrf.design/tokens.html#space)... but wanted to at least have a base consistent style for it.